Energy Saver/Services/ChartService.cs

```diff
@@ -0,0 +1,56 @@
+﻿using ChartJSCore.Helpers;
+using ChartJSCore.Models;
+using Energy_Saver.Model;
+
+namespace Energy_Saver.Services
+{
+    public class ChartService : IChartService
+    {
+        public Chart CreateChart()
```

Comment:
Chart data should be changed. Its very difficult to understand. Maybe change x axis to time(years and months) and y axis to total amount of all taxes at a given month (electricity + water + heating + gas)?